contrib/drivers/ceph/ceph.go

```diff
@@ -440,6 +498,7 @@ func (d *Driver) ListPools() ([]*model.StoragePoolSpec, error) {
 			TotalCapacity:    (p.Stats.BytesUsed + p.Stats.MaxAvail) >> sizeShiftBit,
 			FreeCapacity:     p.Stats.MaxAvail >> sizeShiftBit,
 			StorageType:      d.conf.Pool[p.Name].StorageType,
+			Extras:           extras,
```

Comment:
@imquanquan thanks,  that's a bug. To fixed the bug ,you just need to s/extras/d.conf.Pool[p.Name].Extras, and remove all other codes. propertes in extras.Advanced  is not useless right now.

Reply:
Done.